File: MPI/mpicheck/tests/functions/1.0/MPI_Waitall.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2(10,10)
       INTEGER var_3
        call mpi_waitall(var_0, var_1, var_2, var_3)
       call pmpi_waitall(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Waitall.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Request var_1[2];
    MPI_Status var_2[2];
    int ret;
    /* calls */
    ret = MPI_Waitall(var_0, var_1, var_2);
    ret = PMPI_Waitall(var_0, var_1, var_2);
    return 0;
}
